doc/contributing/primordials.md

```diff
@@ -106,6 +106,7 @@ performance of code in Node.js.
 
 * Methods that mutate the internal state of arrays:
   * `ArrayPrototypePush`
+  * `ArrayPrototypePushApply`: also fails with a RangeError on large arrays
```

Comment:
That’s not the correct place to document, as this is not a performance issue IIUC